plugins/parsers/xpath/README.md

```diff
@@ -8,12 +8,12 @@ For supported XPath functions check [the underlying XPath library][xpath lib].
 
 ## Supported data formats
 
-| name                                    | `data_format` setting | comment |
-| --------------------------------------- | --------------------- | ------- |
-| [Extensible Markup Language (XML)][xml] | `"xml"`               |         |
-| [JSON][json]                            | `"xpath_json"`        |         |
-| [MessagePack][msgpack]                  | `"xpath_msgpack"`     |         |
-| [Protocol buffers][protobuf]            | `"xpath_protobuf"`    | [see additional parameters](protocol-buffers-additiona-settings)|
```

Comment:
@srebhan did you mean to include another doc for protobuf back when you added support?

Reply:
To be honest this should refer to the subsection below (line 18) but it seems I do have a typo there (`protocol-buffers-additiona-settings` vs the correct `protocol-buffers-additional-settings`)... :-(